### ISSUE TITLE
ModuleNotFoundError: No module named 'sorl'

### ISSUE THREAD

**Issue by autoferrit** [cookiecutter/cookiecutter-django#2475]
## What happened?
On a fresh copy of this project, I get an error `ModuleNotFoundError: No module named 'sorl'`. 
```
➜ cookiecutter https://github.com/pydanny/cookiecutter-django

project_name [My Awesome Project]: demo
project_slug [demo]: 
description [Behold My Awesome Project!]: asdf
author_name [Daniel Roy Greenfeld]: Shawn McElroy
domain_name [example.com]: domain.com
email [shawn-mcelroy@example.com]: my@domain.com
version [0.1.0]: 
Select open_source_license:
1 - MIT
2 - BSD
3 - GPLv3
4 - Apache Software License 2.0
5 - Not open source
Choose from 1, 2, 3, 4, 5 [1]: 5
timezone [UTC]: America/Los_Angeles
windows [n]: 
use_pycharm [n]: y
use_docker [n]: y
Select postgresql_version:
1 - 11.3
2 - 10.8
3 - 9.6
4 - 9.5
5 - 9.4
Choose from 1, 2, 3, 4, 5 [1]: 1
Select js_task_runner:
1 - None
2 - Gulp
Choose from 1, 2 [1]: 
Select cloud_provider:
1 - AWS
2 - GCP
3 - None
Choose from 1, 2, 3 [1]: 2
use_drf [n]: y
custom_bootstrap_compilation [n]: 
use_compressor [n]: y
use_celery [n]: 
use_mailhog [n]: 
use_sentry [n]: y
use_whitenoise [n]: 
use_heroku [n]: 
Select ci_tool:
1 - None
2 - Travis
3 - Gitlab
Choose from 1, 2, 3 [1]: 3
keep_local_envs_in_vcs [y]: 
debug [n]: 
 [SUCCESS]: Project initialized, keep up the good work!

➜ pyenv virtualenv 3.8.1 demo

➜ pyenv versions                
  system
  3.7.6
* 3.8.1 (set by /home/shawn/.pyenv/version)
  3.8.1/envs/demo
  demo

➜ pyenv shell demo

➜ pip install -r requirements/local.txt
...
Successfully installed ...

➜ ./manage.py                               
Traceback (most recent call last):
  File "./manage.py", line 30, in <module>
    execute_from_command_line(sys.argv)
  File "/home/shawn/.pyenv/versions/demo/lib/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/shawn/.pyenv/versions/demo/lib/python3.8/site-packages/django/core/management/__init__.py", line 357, in execute
    django.setup()
  File "/home/shawn/.pyenv/versions/demo/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/shawn/.pyenv/versions/demo/lib/python3.8/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/home/shawn/.pyenv/versions/demo/lib/python3.8/site-packages/django/apps/config.py", line 116, in create
    mod = import_module(mod_path)
  File "/home/shawn/.pyenv/versions/3.8.1/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sorl'
```

## What should've happened instead?
I should be able to run manage.py and run the application without errors for the first time

## Steps to reproduce
(shown above)

## System
```
➜ uname -a
Linux shawn-pc 5.4.18-1-MANJARO #1 SMP PREEMPT Thu Feb 6 11:41:30 UTC 2020 x86_64 GNU/Linux

```

**Comment by autoferrit** [comment]
If I install `pip install sorl-thumbnails` I still get another error along the same lines
```
ModuleNotFoundError: No module named 'business'
```
So it seemed like it was some other issue

**Comment by AmiraliMesbah** [comment]
hi i got this error in my project. can enybody help me to fix this error??, thnaks


ModuleNotFoundError: No module named 'sorl'

**Comment by AtusaFa1** [comment]
Hello everyone I have a problem. I installed sorl-thumbnail in my pycharm project with pip install sorl-thumbnail and I put 'sorl.thumbnail', in my INSTALLED_APPS settings . but I have this error again ModuleNotFoundError: No module named 'sorl'

sorl thumbnail is installed in my pip list and in my interpreter settings. Even I changed my django from 4.2.2 to  4.1.7 but it doesn't work . I saw one friend said in stack overflow install "pip install sorl-thumbnail django-haystack pysolr" and I did it but I have this error yet . I really thank you if someone can help me. sorry if my Enghish language is not well . Thank you

#python #Django #django

**Comment by browniebroke** [comment]
Are you using Docker? If so, please check the relevant section of our docs on how to ad a 3rd party package: https://cookiecutter-django.readthedocs.io/en/latest/developing-locally-docker.html#add-3rd-party-python-packages

If you're not using Docker, make sure that the package is installed in the correct environment. This is not a problem specific to Cookiecutter Django, it's about using Python virtualenv.

**Comment by AtusaFa1** [comment]
I installed it in my virtual env . I don't use of Docker. I have ModuleNotFoundError: No module named 'sorl'. I don't know what to do, I searched on internet but not fount my answer! 
 `
(test) PS C:\Users\ASUS\PycharmProjects\course\website> pip list                  
Package                 Version
----------------------- ---------
asgiref                 3.7.2
certifi                 2023.7.22
idna                    3.4
kavenegar               1.1.2
Pillow                  10.0.0
pip                     23.2.1
pysolr                  3.9.0
requests                2.31.0
setuptools              65.5.0
six                     1.16.0
sorl-thumbnail          12.9.0
sqlparse                0.4.4
suds-py3                1.4.5.0
tzdata                  2023.3
urllib3                 2.0.4

 `
